### PR TITLE
Server properties mappings

### PR DESCRIPTION
**Pull request by maxpowa**
For the server impls and the properties loader itself.

Tried to stick to is/shouldXXX as much as possible for booleans, not sure which is best.

### CODE REVIEW

File: mappings/net/minecraft/server/MinecraftServer.mapping [hunk 1]
```diff
@@ -23,37 +23,49 @@ CLASS net/minecraft/server/MinecraftServer net/minecraft/server/MinecraftServer
...
 	METHOD aJ getLogger ()Lorg/apache/logging/log4j/Logger;
+	METHOD a_ setResourcePack (Ljava/lang/String;Ljava/lang/String;)V
 	METHOD aa isDedicated ()Z
 	METHOD ab isOnlineMode ()Z
 	METHOD ac shouldSpawnAnimals ()Z
 	METHOD ad shouldSpawnNpcs ()Z
+	METHOD ae isUsingNativeTransport ()Z
 	METHOD af isPvpEnabled ()Z
 	METHOD ag isFlightEnabled ()Z
+	METHOD ah isCommandBlocksEnabled ()Z
```
Comment (copygirl) [inline mappings/net/minecraft/server/MinecraftServer.mapping]: `areCommandBlocksEnabled`

File: mappings/net/minecraft/server/dedicated/DedicatedServer.mapping [hunk 2]
```diff
@@ -1,7 +1,30 @@
...
+	FIELD t gameMode Lnone/ais;
+	METHOD <init> (Ljava/io/File;Lnone/pf;Lcom/mojang/authlib/yggdrasil/YggdrasilAuthenticationService;Lcom/mojang/authlib/minecraft/MinecraftSessionService;Lcom/mojang/authlib/GameProfileRepository;Lnone/mn;)V
+		ARG 2 authService
+	METHOD B exit ()V
+	METHOD E isNetherAllowed ()Z
+	METHOD U isMonsterSpawningEnabled ()Z
 	METHOD Z isSnooperEnabled ()Z
+	METHOD a setGameMode (Lnone/ais;)V
+		ARG 0 gameMode
+	METHOD aD getMaximumWorldSize ()I
```
Comment (copygirl) [inline mappings/net/minecraft/server/dedicated/DedicatedServer.mapping]: Does this method affect the world border? If so, should it be `getMaxWorldBorderRadius` instead? (**edit:** Added `Max` to suggested method name.)

File: mappings/net/minecraft/server/dedicated/DedicatedServer.mapping [hunk 3]
```diff
@@ -1,7 +1,30 @@
...
+	METHOD aF getNetworkCompressionThreshold ()I
+	METHOD aP getMaximumTickTime ()J
 	METHOD aa isDedicated ()Z
+	METHOD ae isUsingNativeTransport ()Z
+	METHOD ah isCommandBlocksEnabled ()Z
```
Comment (copygirl) [inline mappings/net/minecraft/server/dedicated/DedicatedServer.mapping]: `areCommandBlocksEnabled`

**Comment by asiekierka** [comment]
Merge conflicts have appeared; also, LGTM beyond copygirl's input

**Comment by shadowfacts** [comment]
Other than the incorrect is/are, LGTM

**Comment by maxpowa** [comment]
Thanks for the input, I'll switch to 'are'. Again, something that I was questioning about in the IRC channel but no substantial responses, timezones and such.

**Comment by asiekierka** [comment]
Sure. This is why you make a pull request.

**Comment by maxpowa** [comment]
Should be fixed up now, changed method names per @copygirl's suggestions.

**Pull request merged**